src/main_WorldViewer.cpp

```diff
@@ -79,7 +80,12 @@ class REGothWorldViewer : public REGoth::REGothEngine
 
     HCharacter hero = heroSO->getComponent<Character>();
 
+    mMainCamera->SO()->setActive(false);
+    mMainCamera->SO()->setActive(true);
```

Comment:
Why is this necessary?

Reply:
I thought I added a comment here. There is an issue about the camera not moving and this is the workaround.